### ISSUE TITLE
dnsi vs. idns

### ISSUE THREAD

**Issue by paulehoffman** [NLnetLabs/dnsi#13]
The documentation talks about idns and dnsi. Pick one. :-)

Also, the man page talks about "dnsi-lookup" and "dnsi-query", which don't seem to exist yet.

**Comment by AlexanderBand** [comment]
Thanks for giving `dnsi` a spin and providing us with feedback Paul! The project is now five days old so things are a bit in flux, but we definitely decided on `dnsi`. 

The `lookup` command was built and merged yesterday, so it isn't part of the 0.1.0 release yet. Please try installing from the `main` branch like so:

``` bash
cargo install --force --git https://github.com/NLnetLabs/dnsi --branch main
```

Then try:

``` bash 
dnsi lookup nlnetlabs.nl icann.org blablablablabla.bla 185.199.110.153 192.168.5.5 
```

**Comment by paulehoffman** [comment]
FWIW, I prefer "dnsi" instead of "idns", so I'm happy here. :-)
